com.insynchq.Insync.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.insynchq.Insync</id>
+  <name>Insync</name>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
link to license?

Reply:
I couldn't find a page, from them where this is stated. At least not in the sense of a code license.

Reply:
what about the EULA that they force you to agree?

Reply:
Added